src/main/java/org/assertj/core/api/ObjectEnumerableAssert.java

```diff
@@ -1397,7 +1397,7 @@
    *                                         character -&gt; assertThat(character.isMortal()).isTrue(),
    *                                         character -&gt; assertThat(character.getName()).isEqualTo("Legolas"));
    *
-   * // you can specify more that one assertion per requirements
+   * // you can specify more tha one assertion per requirement
```

Comment:
```suggestion
   * // you can specify more than one assertion per requirement
```

Reply:
Doh! 🙈 Reminder to myself: Before submitting, check that not only have the correct words been modified, but that the words have been modified correctly.

Thanks for proofreading. 😁